cuda/test/matrix/sellp_kernels.cpp

```diff
@@ -295,7 +275,7 @@ TEST_F(Sellp, AdvancedApplyToComplexIsEquivalentToRef)
 
 TEST_F(Sellp, ConvertToDenseIsEquivalentToRef)
 {
-    set_up_apply_matrix();
+    set_up_apply_matrix(64);
 
     auto dense_mtx = gko::matrix::Dense<>::create(ref);
```

Comment:
These empty lines should also be deleted (and in the following tests) since the setup and the following allocations are part of the same Allocation block:
```suggestion
    set_up_apply_matrix(64);
    auto dense_mtx = gko::matrix::Dense<>::create(ref);
```